Extensions/xkit_patches.js

```diff
@@ -124,33 +124,74 @@ XKit.tools.parse_version = function(versionString) {
 	return version;
 };
 
+var blogs_get = document.createElement("script");
+blogs_get.textContent = "("+(function(){
+	var blogs = [];
+	var models = Tumblr.dashboardControls.allTumblelogs;
+
+	models.filter(function(model){
```

Comment:
If this being inserted into DOM context is to get around the property access violation error it's much simpler to just rewrite this in terms of a for loop with standard numerical indexing which will not trigger the error.